README.md

```diff
@@ -231,11 +231,12 @@ We are building on top of Acorn, however, so that we can contribute back and hel
 
 ### What ECMAScript features do you support?
 
-Espree supports all ECMAScript 2023 features and partially supports ECMAScript 2024 features.
+Espree supports all ECMAScript 2023 features and partially supports ECMAScript 2024 and ECMAScript 2025 features.
 
-Because ECMAScript 2024 is still under development, we are implementing features as they are finalized. Currently, Espree supports:
+Because ECMAScript 2024 and ECMAScript 2025 are still under development, we are implementing features as they are finalized. Currently, Espree supports:
```

Comment:
ES2024 will be under development until the end of June, right?

Reply:
I believe that ECMAScript 2024 features are already finalized and that publishing the final spec in June is just a formality. So I think it's already safe to say that "Espree supports all ECMAScript 2024 features and partially supports ECMAScript 2025 features." but would like @nzakas's opinion too.

Reply:
And I think it can only happen that some features get dropped from the final version, while no new features can be added at this point, so seems safe to say that we fully support ECMAScript 2024.

Reply:
TC39 is meeting this week, so everything for 2024 should be finalized today.

Reply:
Thank you for your comments! I've updated the README.